client/src/views/users/InvitationsListPopover.vue

```diff
@@ -0,0 +1,172 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
+
+<template>
+  <div class="invitations-list-container">
+    <div class="invitations-list">
+      <div class="invitations-list-header">
+        <ion-text class="invitations-list-header__title title-h4">
+          {{ $t('invitationList.title') }}
+        </ion-text>
+        <span class="invitations-list-header__counter body-sm">
+          {{ invitations.length }}
+        </span>
+        <ion-button
+          @click="onInviteClick"
+          class="invitations-list-header__button"
+          size="small"
+        >
+          <ion-icon
+            slot="start"
+            :icon="personAdd"
+            class="button-icon"
+          />
+          {{ $t('invitationList.invite') }}
+        </ion-button>
+      </div>
+      <ion-list class="invitations-list-content">
+        <div
+          class="invitations-list-content__empty"
+          v-if="invitations.length === 0"
+        >
+          <ion-text class="body-lg">
+            {{ $t('invitationList.noInvitations') }}
+          </ion-text>
+        </div>
+        <invitation-popover-item
+          v-for="invitation in invitations"
+          :invitation="invitation"
+          :key="invitation.claimerEmail"
+          @cancel="cancelInvitation"
```

Comment:
It would be nice to have this popover in the `InvitationCard` and `InvitationListItem` reject buttons.

Reply:
Those components will be removed.